skietbaan_fe/src/components/Login.js

```diff
@@ -30,29 +29,27 @@ class App extends Component {
       stateUpdate.invalidPassword= true;
       isValid = false;
     };
-    
```

Comment:
Please re-add these lines in between code sections. Whitespace is only invalid when there is more than one empty line or at the start or end of a function but you are allowed a line in between if statements for example.